row_050_2/row_050_2_unsafe.py

```
@@ -0,0 +1,344 @@
+
+def newc():
+    def inr1():
+        var1 = var.get()
+        connection = pymysql.connect(host='localhost', user='root', password='', db='class')
+        cursor = connection.cursor()
+        q = ("create table " + str(var1) + "(Name text,regno int,attendence text, PRIMARY KEY(regno))")
+        try:
+            cursor.execute(q)
+            connection.commit()
+            l.config(text="Successful",bg="GREEN",fg="Dark GREEN")
+        except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)